monitoring/files/docker-compose.yml

```diff
@@ -0,0 +1,112 @@
+version: '3.7'
+
+services:
+  grafana:
+    image: grafana/grafana:8.4.4
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "20m"
+        max-file: "10"
+    networks:
+      - monitoring
```

Comment:
Does this network need to be defined?

Reply:
Yeah so the way I do it is just define the network directly in the same compose file, see https://docs.docker.com/compose/networking/